.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Grain Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "The tag of release"
+        required: true
+
+jobs:
+  release-updates:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
+
+      - name: Replace download links
+        run: |
+          sed -E -i 's/(https:\/\/github\.com\/grain-lang\/grain\/releases\/download\/)[^\/]+(\/)/\1${{ github.event.inputs.tag }}\2/g' src/getting_grain.md
```

Comment:
@peblair @ospencer sanity check on this bash-fu

Reply:
I'd streamline this to be a `'s|foo|bar|g'`-style pattern, just to avoid all of these escapes. Something like (untested):
```
's|(https://github\.com/grain-lang/grain/releases/download/)[^/]+(/)|\1${{ github.event.inputs.tag }}\2|g'
```

Reply:
Absolute champion: https://github.com/phated/grain-lang.org/commit/61bdad944998ce3992e2ea0e7b40efacc51687e6